app/views/admin/sites/_sites_neighbourhood_fields.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="nested-fields">
+    <%= f.input :name, collection: options_for_sites_neighbourhoods, include_blank: false,
+                input_html: { class: 'form-control select2' } %>
+    <%= f.hidden_field :relation_type, value: 'Secondary' %>
+    <%= link_to_remove_association 'Remove Secondary Neighbourhood', f, class: "btn btn-danger" %>
```

Comment:
let's tidy these up a bit with buttons to right of ward names, prob need to check bootstrap docs